README.md

```diff
@@ -24,34 +24,44 @@ cmake ..
 make -j8
 make test
 ```
+
+For choosing the compiler, use the standard CMake techniques, e.g. setting the environment variables
+```
+CXX=`which g++` # full path to the C++ compiler
+CC=`which gcc` # full path to theC compiler
+FC=`which gfortran` # full path to theFortran compiler
+CUDACXX=`which nvcc` # full path to NVCC
+CUDAHOSTCXX=`which g++` # full path to the C++ compiler to be used as CUDA host compiler
+```
+
 ##### Requirements
 
 - Boost (1.65.1 or later)
 - CMake (3.14.5 or later)
 - CUDA Toolkit (9.0 or later, optional)
+- MPI (optional, CUDA-aware MPI for the GPU communication module `gcl_gpu`)
 
 ### Supported compilers
 
 The GridTools libraries are currently nightly tested with the following compilers on [CSCS supercomputers](https://www.cscs.ch/computers/overview/).
 
-| Compiler | Backend | Tested on |
-| --- | --- | --- |
-| NVCC 10.1 with GNU 6.2 | gpu | Piz Daint |
-| GNU 7.3.0 | cpu_kfirst, cpu_ifirst | Piz Daint |
-| Clang 7.0.1 | cpu_kfirst, cpu_ifirst | Piz Daint |
-
-##### Known issues
-
-- Intel is able to compile GridTools code, but depending on user code, might have severe performance problems compared to GNU- or Clang-compiled code.
+| Compiler | Backend | Tested on | Comments
+| --- | --- | --- | --- |
+| Cray clang version 9.0.2 | all backends | Piz Daint | with flags `-fno-cray-gpu -fno-cray`, P100 GPU
+| GNU 7.3.0 + NVCC 10.1 | all backends | Piz Daint | P100 GPU |
+| Clang 7.0.1 + NVCC 10.1 | all backends | Piz Daint | GPU compilation in NVCC-CUDA mode, P100 GPU |
+| GNU 8.3.0 + NVCC 10.1 | all backends | Tsa | V100 GPU |
+| HIP Clang (TODO) | all backends | Ault | TODO GPU |
```

Comment:
@fthaler Please fill in a description of the compiler (maybe put a link to the docs where we mention how to build this compiler)

Reply:
Providing a link is a bit tricky, I think in the docs we mention https://github.com/ROCm-Developer-Tools/HIP/blob/master/INSTALL.md somewhere. But actually I had to use the master-next branch of HIP to make it work (found this info in some issue or PR comment) and things seem to change fast… Probably we can just rely on people asking in case somebody is trying without success?

Reply:
Makes sense.